packages/strapi-parts/src/TextInput/TextInput.stories.mdx

```diff
@@ -11,11 +12,32 @@ import { Tooltip } from '../Tooltip';
 
 # TextInput
 
-This is the doc of the `TextInput` component
+TextInputs are inputs that allow users to add or edit text. They support several text formats including numbers.
 
-## TextInput
+**Best practices**
 
-Description...
+- Icons are always on the right side within the input (writing area) itself, never on the left side.
+- TextButton component should always be above the input. 
```

Comment:
I don't know what is this TextButton thing 🤔 . But people don't have access to this from outside the component

Reply:
Oh okay! Thanks!